### DISCUSSION TITLE
Emojis

### DISCUSSION THREAD

**Discussion by beto0890** [USU-ITLS-x265-23-summer/class-repo#73] [Q&A] [answered]
Hello all,

2 questions!

Our "readme.md" is something different from our (week 1, task 3, assignment 4, "personal page)" or are they the same thing?

How do I find the emojis to use on the "readme.md"

**Comment by jakegros** [comment, marked as answer]
It is something different.  The readme.md appears on your GitHub profile, while the Week 1 task was within our course.
You can bring up the emojis menu to add in by:
On Windows:  Windows logo key + . (period)  
On Mac:  control + command + space

It may take a few attempts. I hope this is what you were going after.

**Comment by jakegros** [comment]
FYI, here is some info on using emojis in general
[GitHub emoji cheatsheet](https://github.com/tiimgreen/github-cheat-sheet#emojis )

**Reply by beto0890** [reply]
Thanks so much!
